tests/gnrc_tcp/README.md

```diff
@@ -18,6 +18,11 @@ in the tests directory.
     This test covers receiving of a byte stream from the host system. The received data is
     causing window opening and closing as well as data transmission over multiple packets.
 
+6) 06-receive_data_closed_conn.py
+    This test covers accessing received data after receiving a FIN packet. If the connection was closed
+    by the peer, a call to gnrc_tcp_recv must return directly with all currently received data
+    or zero if there is no data. The function must return immediatly dispite any given timeout.
```

Comment:
Please put the new tests into a separate commit.

Reply:
Why. Does it not make sense to supply a test that verifies a change within the same PR?

Reply:
Same PR yes, but they should be a separate commit, as they are not part of the fix themselves.